website/docs/tasks.md

```diff
@@ -167,16 +145,16 @@ This feature is currently experimental.
 :::
 
 If you're running on Linux or Unix (including macOS) then if Graphile Worker
-finds an executable file inside of `tasks/` it will create a task executor for
-it. When a task of this kind is found, Graphile Worker will execute the file
-setting the relevant environmental variables and passing in the payload
+finds an executable file inside of the `taskDirectory` it will create a task
+executor for it. When a task of this kind is found, Graphile Worker will execute
+the file with the relevant environment variables and pass in the payload
 according to the encoding. If the executable exits with code `0` then Graphile
-Worker will see this as success, all other exit codes are seen as failure.
+Worker will see this as success. All other exit codes are seen as failure.
 
-### Environmental variables
+### Environment variables
```

Comment:
Wow; I’ve been wrong about this my entire career 🤯

Reply:
I had to google this to see if this is a UK thing or something 🙂